packages/web-forms/src/components/controls/InputText.vue

```diff
@@ -5,7 +5,7 @@ import ControlLabel from '../ControlLabel.vue';
 
 const props = defineProps<{question: StringNode}>();
 
-const setValue = (value: string) => {
+const setValue = (value = '') => {
```

Comment:
❓ can you please explain this change

Reply:
![image](https://github.com/getodk/web-forms/assets/199830/e77ece41-365a-4c27-8df2-14a9ec741f5a)